Updates to Collections + Sync [DOC-1038] (#7482)
segmentio/segment-docs: src/connections/sources/catalog/cloud-apps/hubspot/index.md

Our sync component uses an upsert API, so the data in your warehouse loaded using sync will reflect the latest state of the corresponding resource in HubSpot. For example, if `deals` goes from `open` to `closed` between syncs, on its next sync that deal's status will be `closed`.

> The source syncs and warehouse syncs are independent processes. Source runs pull your data into the Segment Hub, and warehouse runs flush that data to your warehouse.  You can set the start date of the first sync and sources will sync with Segment every 3 hours. Depending on your Warehouses plan, we will push the Source data to your warehouse on the interval associated with your billing plan.

Reviewer: Can you set the date of your first source sync or warehouse sync (or both?)

When possible, the docs team likes to refer to what Segment can do in present tense rather than in future tense (although the existing docs page does have future tense on it!) 

Also, the docs team generally refers to stuff Segment can do as "Segment does x" rather than "We do x", mostly because we've got so many third-party partners that we host docs for and it can get confusing as to who has what capabilities unless we're super direct about it!

```suggestion
The source syncs and warehouse syncs are independent processes. Source runs pull your data into the Segment Hub, and warehouse runs flush that data to your warehouse.  You can set the start date of the first sync. After the first sync, sources sync with Segment every 3 hours. Depending on your Warehouses plan, Segment pushes the Source data to your warehouse on the interval associated with your billing plan.
```

Reviewer: yes of course!

## Collections

> Collections are the groupings of resources we pull from your source. You can select which collections will be included in your sync. In your warehouse, each collection gets its own table.

Reviewer: ```suggestion
Collections are the groupings of resources we pull from your source. You can select which collections are included in your sync. In your warehouse, each collection gets its own table.
```
